### ISSUE TITLE
Setting initial state belief in TabularPOMDP?

### ISSUE THREAD

**Issue by ShawnGeller** [JuliaPOMDP/POMDPModels.jl#76]
Hello, thank you for making this package! I wanted to know how to set the initial state belief in a TabularPOMDP. It looks like currently it's hard coded to be uniform over states. I'm not very familiar with Julia, so I tried extending this myself but had some trouble.

**Comment by zsunberg** [comment]
Yes, you are correct. For now, as a hack, you can override it yourself by overriding the `initialstate` function, for example:
```julia
POMDPs.initialstate(p::TabularPOMDP) = POMDPModelTools.Deterministic(1)
```
Eventually, we should put another field in the TabularPOMDP struct so that people can set it via the constructor. PRs welcome :)

**Comment by ShawnGeller** [comment]
Got it, thanks for the quick hack!
